player/cache.go

```diff
@@ -94,10 +94,41 @@ func InitFSCache(opts *FSCacheOpts) (ChunkCache, error) {
 			MetCacheSize.Set(float64(c.Size()))
 		}
 	}()
+	go func() {
+		Logger.Infoln("restoring cache in memory...")
+		err := c.reloadCache()
+		if err != nil {
+			Logger.Errorf("failed to restore cache in memory: %s", err.Error())
+		}
+		Logger.Infoln("done restoring cache in memory")
+	}()
 
 	return c, nil
 }
 
+func (c *fsCache) reloadCache() error {
```

Comment:
This would benefit from tests and can be easily covered, check `cache_test.go` for examples

Reply:
yeah I didn't mess with tests, maybe I can take a quick look now